test/unit/mdc-drawer/dismissible.foundation.test.js

```diff
@@ -179,8 +186,7 @@ test(`#isClosing returns true when it has ${cssClasses.CLOSING} class`, () => {
 });
 
 test(`#isClosing returns false when it lacks ${cssClasses.CLOSING} class`, () => {
-  const {foundation, mockAdapter} = setupTest();
-  td.when(mockAdapter.hasClass(cssClasses.CLOSING)).thenReturn(false);
+  const foundation = new MDCModalDrawerFoundation();
```

Comment:
Why was this test changed to instantiate a subclass?

Reply:
To increase code coverage. `setupTest()` passes an argument to the foundation constructor, so Istanbul thinks we need to test the constructor _without_ any arguments.

I'll see if I can revert this and use an `/* istanbul ignore next */` comment instead.

Reply:
Looks like this change is no longer necessary because I already added
the `/* istanbul ignore next */` comment. Reverted.